### DISCUSSION TITLE
LDAP User auth and user key for secret store

### DISCUSSION THREAD

**Discussion by greinick** [netbox-community/netbox#10004] [Q&A]
Hi, I added LDAP auth to my netbox installation which works fine. But I cant manage to generate or add a user key for the secret store.

Any how to available?

Netbox 3.2.8

Thanks for hints /Götz

**Comment by candlerb** [comment]
Which "secret store" are you referring to? The Secrets function was removed from Netbox in [v3.0.0](https://github.com/netbox-community/netbox/releases/tag/v3.0.0). Are you talking about this [plugin](https://github.com/dansheps/netbox-secretstore/)? If so, perhaps you could explain the sequence of steps you're taking to try to add a user key, and what response you see when you do them?

It seems unlikely that how you authenticate to Netbox would make any difference to the plugin functionality, but are you seeing any LDAP-related errors?

**Comment by greinick** [comment]
Hi and thanks for your feedback. I was lookin into the LDAP users profile when logged in as described here:

https://github.com/DanSheps/netbox-secretstore/blob/develop/docs/models/userkey.md

I looked also into the admin panel, but dint see an option to generate a pair for the ldap user....

Though is there an other way? Or did I miss something? Enabling the log file as described in the docs did not work for me - an other story ... - 


/G
